client/state/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import thunkMiddleware from 'redux-thunk';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import sharing from './sharing/reducer';
+import sites from './sites/reducer';
+import ui from './ui/reducer';
+
+/**
+ * Module variables
+ */
+const reducer = combineReducers( {
+	sharing,
+	sites,
+	ui
+} );
+
+export function createReduxStore() {
```

Comment:
@aduth, @mtias : I will change this in #1393 to export not the function, but singleton - already instantiated store.
context.store is not helpful at all and deep in some component that is not reduxified yet I need to call notices.

I dont want to reduxify the whole app yet, but notices are created from virtually anywhere, so I kinda need something to import to actually create a notice.